### ISSUE TITLE
\list does not work and \l only has one column

### ISSUE THREAD

**Issue by Flimm** [dbcli/pgcli#674]
## Description

I expected `\list` to return a list of databases, instead it gives this error:

```
syntax error at or near "\"
LINE 1: \list
        ^

Time: 0.001s
```

If I use `psql`, `\list` behaves like `\l`, it gives a list of databases with these column headers:

```
                                           List of databases
          Name          |   Owner    | Encoding |   Collate   |    Ctype    |     Access privileges
------------------------+------------+----------+-------------+-------------+---------------------------
 example                | example    | UTF8     | en_GB.UTF-8 | en_GB.UTF-8 |
```

In pgcli, `\l` only has one column:

```
+------------------------+
| datname                |
|------------------------|
| example                |
+------------------------+
```

I think pgcli should be have like psql: both `\list` and `\l` should be supported, and it should output all the columns "name", "owner", "encoding", etc.

## Your environment

I'm using version: 1.3.1 of pgcli and comparing it to version 9.6.2 of psql.

**Comment by amjith** [comment]
Oooh. This must be new behavior for psql. It used to show only one column. You're right we should try to duplicate this behavior in pgcli. 

Thank you for the issue. 👍

**Comment by fpietka** [comment]
@amjith I really can't remember when psql showed only one column, but at least in 9.5+ it aways shows all those columns. The query I added in #746 is the query to list all those, so we can use it (maybe replace the old one?)

note that with psql `\l+`/`\list+` also returns more columns: `Size`, `Tablespace` and `Description`

**Comment by amjith** [comment]
I'd be glad to merge a PR with the new query you've got. Would you like to make a PR or shall I create one on your behalf?

**Comment by fpietka** [comment]
I'll be glad to do it.

**Comment by fpietka** [comment]
Fixed since 1.8.2
